_sass/_layout.scss

```diff
@@ -16,3 +16,14 @@ footer {
   border-top: 1px solid $divider-color;
   padding: $base-spacing 0;
 }
+
+.bio {
+  display: inline-block;
```

Comment:
We could wrap all the bio's in a parent flex container that should automatically resize the images based on the screen-width. Working on a codepen for an example...